packages/vanilla-renderers/src/cells/TextCell.tsx

```diff
@@ -50,7 +50,9 @@ export const TextCell = (props: CellProps & VanillaRendererProps) => {
   const appliedUiSchemaOptions = merge({}, config, uischema.options);
   return (
     <input
-      type='text'
+      type={
+        appliedUiSchemaOptions.format ? appliedUiSchemaOptions.format : 'text'
+      }
```

Comment:
```suggestion
      type={appliedUiSchemaOptions.format === 'password' ? 'password' : 'text'}
```

Reply:
I would prefer only passing through the password as in React Material. Other types might be incompatible. I would have applied the change myself but maintainer editing was turned off for this PR